components/wpt-results.html

```diff
@@ -125,6 +128,10 @@
             observer: '_queryChanged'
           },
           specs: {
```

Comment:
General naming comment: I think that using the word "specs" isn't quite correct here; top-level directories in wpt are supposed to correspond to specifications, but they are not themselves specs.

"Specs" here does refer to top-level directories in wpt, right? I do appreciate that "specs" is nice and short and generally gives the right idea though. How do you feel about this?